.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,50 @@
+name: Commitlint
+
+on:
+  push:
+    branches:
+      - 'push-action/**'
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+concurrency:
+  group: commitlint-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint-commits:
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install commitlint tools
+        run: |
+          npm install -g @commitlint/config-conventional @commitlint/cli
```

Comment:
Can we run `npx` instead of global installation?

Reply:
We need to install the whole package because we need to use the `@commitlint/config-conventional` in our config file.
Like the Go SDK project, this script is only run in the ci environment; we don't need to guard the commit message in our local dev machine.
- PR in the Go SDK: https://github.com/logto-io/go/pull/22
- Related Discussion: https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1661486799784279

Reply:
Good call; after testing, it works.